### PR TITLE
Add Github action template, closes #1

### LINKED ISSUE #1: Make template for GitHub actions
😎

### PR DESCRIPTION
**Pull request by abskmj**

### CODE REVIEW

File: generator/github_template/github-workflow.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+name: Workflow
+
+# trigger when new commits are pushed to master or main branch
+on:
+  push:
+    branches: [ master, main ]
```
Comment (P0ppoff) [inline generator/github_template/github-workflow.yml]: `main` branch seems real specific and not there when you use the plugin. I suggest to just trigger on `master`

File: generator/github_template/github-workflow.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+      with:
+        registry-url: https://registry.npmjs.org/
+    - run: yarn install
+    - run: yarn lint
+    - run: yarn test:unit
+    - run: yarn build
+    - name: Publish Artifact
+      run: yarn publish
+      env:
+        NODE_AUTH_TOKEN: ${{secrets.YARN_TOKEN}}
```
Comment (P0ppoff) [inline generator/github_template/github-workflow.yml]: It's a common step for a package, but the plugin is used with app. We don’t need to publish it. You can erase these lines. 

The last step for other agents is to save the artifact `/dist` to make it accessible and downloadable. Can you make change in this way ?

File: generator/github_template/github-workflow.yml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install Node.js
+      uses: actions/setup-node@v1
+      with:
+        registry-url: https://registry.npmjs.org/
```
Comment (P0ppoff) [inline generator/github_template/github-workflow.yml]: Can we use node 12 ? And don't add default param for the registry ?

**Review by P0ppoff — CHANGES_REQUESTED** [review]

File: generator/github_template/github-workflow.yml (hunk 2 above)
Comment (abskmj) [reply generator/github_template/github-workflow.yml]: I misunderstood saving the artifact for publishing to the NPM registry. Let me make the suggested changes.

Comment (abskmj) [reply generator/github_template/github-workflow.yml]: Updated as per your suggestions

**Pull request merged**